src/array_utils.h

```diff
@@ -56,4 +62,106 @@ shape_from_tuple(nb::tuple new_shape, size_t elem_count)
     return new_shape_vec;
 }
 
+/**
+ * @brief Converts the provided axes into a vector of positive indices.
```

Comment:
I wonder if our doc system has support for this. Let's see...

Reply:
I realize that the docs are not build for `array_utils`. It must be added in https://github.com/apytypes/apytypes/blob/main/docs/cpp_api/index.rst (I noted that the rst-file was added, but didn't think of this one.)